src/stylesheets/core/_notifications.scss

```diff
@@ -15,79 +15,22 @@ $uswds-notifications:
-  + "\A - We updated some settings defaults:"
-  + "\A   $theme-color-success-dark: green-cool-50v (was green-cool-50)"
-  + "\A   $theme-color-success-darker: green-cool-60v (was green-cool-80)"
-  + "\A --------------------------------------------------------------------"
-  + "\A 2.11.2:"
-  + "\A - We deprecated the `$theme-site-max-width` variable. We're using"
-  + "\A   `$theme-grid-container-max-width` instead."
-  + "\A - We replaced the `thumb_down_off_alt` icon with `thumb_down_alt`"
-  + "\A   in our default icon sprite."
-  + "\A --------------------------------------------------------------------"
-  + "\A 2.11.0:"
-  + "\A - We updated some settings defaults:"
-  + "\A   $theme-alert-icon-size: 4 (was 5)"
-  + "\A   $theme-table-border-color: default (was 'ink')"
-  + "\A   $theme-table-header-text-color: default (was 'ink')"
-  + "\A   $theme-table-stripe-text-color: default (was 'ink')"
-  + "\A   $theme-table-text-color: default (was 'ink')"
-  + "\A --------------------------------------------------------------------"
-  + "\A 2.10.1:"
-  + "\A - We updated the `$theme-breadcrumb-background-color` default"
-  + "\A   to `default` from `'white'`. "
-  + "\A - We updated the `$theme-alert-icon-size` default to `5` from `4`."
-  + "\A - [Footer] We updated `usa-footer__logo-heading` to use a `p`."
-  + "\A   instead of an `h3`. "
-  + "\A --------------------------------------------------------------------"
-  + "\A 2.10.0:"
-  + "\A - We updated the Spanish language link to USAGov in the identifier. "
-  + "\A   The link should now read 'Visite USAGov en Español'. "
-  + "\A - We improved the accessibility of SVG images in the banner. "
-  + "\A   See the full release notes for more information."
-  + "\A --------------------------------------------------------------------"
-  + "\A 2.9.0:"
-  + "\A - We updated the formatting of `usa-legend`. If you use a visible "
-  + "\A   legend and wish to preserve the old styling, add a "
-  + "\A   `usa-legend--large` modifier to the element."
-  + "\A --------------------------------------------------------------------"
-  + "\A 2.8.0:"
-  + "\A - We updated the text of the gov banner and its settings."
-  + "\A   Please see the full release notes for more information and update"
-  + "\A   your project's banner as soon as possible."
-  + "\A --------------------------------------------------------------------"
-  + "\A 2.7.1:"
-  + "\A - We added `not dead` to our autoprefixer settings and now use a "
-  + "\A   `.browserslistrc` file for these options."
-  + "\A --------------------------------------------------------------------"
-  + "\A 2.7.0:"
-  + "\A - We added new usa-button-group, usa-card, usa-character-count, and"
-  + "\A   usa-combo-box components."
-  + "\A - We updated our guidance and code for numeric fields to use `text`"
-  + "\A   rather than `number` inputs with an `inputmode` of `numeric`."
+  + "\A 2.0.2: We changed the names of some settings and mixins."
```

Comment:
Changed the order here to chronological from reverse-chronological so the most current notification is at the bottom. This means that the most current notification is visible in the terminal window when the compile completes and users don't have to scroll up to see the most current notification